src/logs_sender/send_logs.go

```
@@ -118,6 +118,47 @@ func getJournalLogsWithFilter(l LogsSender, since string, outputFilePath string,
 	return nil
 }
 
+func getDmesgLogs(l LogsSender, outputFilePath string) error {
+	log.Infof("Running dmesg")
+	stderr, exitCode := l.ExecuteOutputToFile(outputFilePath, "dmesg")
+	if exitCode != 0 {
+		err := errors.Errorf(stderr)
+		log.WithError(err).Errorf("Failed to run dmesg command")
+		return err
+	}
+	return nil
+}
+
+func getCoreDump(l LogsSender, outputFilePath string) error {
+	log.Infof("Copy coredump file")
+	cmd := "cat $(sysctl kernel.core_pattern | cut -d '|' -f2 | cut -d ' ' -f1)"
+	stderr, exitCode := l.ExecuteOutputToFile(outputFilePath, "bash", "-c", cmd)
+	if exitCode != 0 {
+		err := errors.Errorf(stderr)
+		log.WithError(err).Errorf("Failed to copy coredump file")
```

Failed to get core dump file. This isn't really copying anything.
Does this file always exist?

---

ExecuteOutputToFile actually copies the output of the command into the specified file, but I can rephrase.
The coredump file should always exist in the host, but I can ignore the errors just to be on the safe side.